variables.tf

```diff
@@ -222,7 +222,7 @@ variable "sysdig_crn" {
 ##############################################################################
 
 variable "existing_key_protect_instance_guid" {
-  description = "The GUID of the Key Protect instance in which the key specified in var.key_protect_key_crn is coming from. Required if var.create_cos_instance is true in order to create an IAM Access Policy to allow Key protect to access the newly created COS instance."
+  description = "The GUID of the Key Protect or Hyper Protect instance in which the key specified in var.key_protect_key_crn is coming from. Required if var.skip_iam_authorization_policy is false in order to create an IAM Access Policy to allow Key protect or Hyper Protect to access the newly created COS instance."
```

Comment:
If we are going with the breaking change in https://github.com/terraform-ibm-modules/terraform-ibm-cos/pull/265#issuecomment-1471869957 then I am OK with using one variable here for both key protect and hyper protect. Otherwise, I think we should split the variable into two variables (create new on `existing_hyper_protect_instance_guid`) so that its obvious the module now supports Hyper Protect. To be discussed internally and finalised this week